### PR TITLE
API: Minor tweaks

### PR DESCRIPTION
**Pull request by danielballan**
See commit messages.

### CODE REVIEW

File: filestore/api.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 from .commands import insert_resource, insert_datum, retrieve, db_disconnect
 from .retrieve import register_handler, deregister_handler
+from .odm_templates import ALIAS
```
Comment (tacaswell) [inline filestore/api.py]: :-1: on exposing `ALIAS` in the api

**Comment by danielballan** [comment]
Note: `ALIAS` is used by databroker (in tests) so I made it part of the public API.

**Comment by tacaswell** [comment]
Why is databroker using ALAIS is

**Comment by danielballan** [comment]
https://github.com/NSLS-II/dataportal/pull/80/files#diff-a6efd325a03666e00319f2a800645e76R38

**Comment by tacaswell** [comment]
What should be done is that the logic in the decorator to connect to the db
should be pulled out into it's own `db_connect` method (which takes the
host, port, dbname as args) and has the ALIAS logic hard coded and then
_that_ function should be exposed out to the public API.

I am more ok with dataportal reaching into non-public API than exposing
ALIAS as part of the api

**Pull request merged**